latest-release.ts

```diff
@@ -0,0 +1,31 @@
+import * as nodegit from "nodegit";
+import * as semver from "semver";
+
+
+export async function checkoutLatestRelease(
+    r: nodegit.Repository|Promise<nodegit.Repository>, dir?: string)
```

Comment:
style nit: once you start line-breaking arguments, I'd break all of them. So `dir` on it's own line.